_includes/content/reqs-kernel.md

```diff
@@ -4,28 +4,16 @@
 > satisfy these. 
 {: .alert .alert-success}
 
-- `ip_set`
-- `ip_tables` (for IPv4)
-- `ip6_tables` (for IPv6)
-- `ipt_REJECT`
-- `ipt_rpfilter`
-- `ipt_set`
-- `nf_conntrack_netlink` subsystem
-- `nf_conntrack_proto_sctp`
-- `sctp`
-- `xt_addrtype`
-- `xt_comment`
-- `xt_conntrack`
-- `xt_icmp` (for IPv4)
-- `xt_icmp6` (for IPv6)
-- `xt_ipvs`,`ipt_ipvs`
-- `xt_mark`
-- `xt_multiport`
-- `xt_rpfilter`
-- `xt_sctp`
-- `xt_set`
-- `xt_u32`
-- `xt_bpf` (for eBPF)
-- `vfio-pci`
-- `ipip` (if using {{site.prodname}} networking in IPIP mode)
-- `wireguard` (if using WireGuard encryption)
+Due to the large number of distributions and kernel version out there, it's hard to be precise about the names of the particular kernel modules that are required to run {{site.prodname}}.  However, in general, you'll need:
+
+- The `iptables` modules (both the "legacy" and "nft" variants are supported). These are typically broken up into many small modules, one for each type of match criteria and one for each type of action.  {{site.prodname}} requires:
+
+  - The "base" modules (including the IPv6 versions if IPv6 is enabled in your cluster).
+  - At least the following match criteria: `set`, `rpfilter`, `addrtype`, `comment`, `conntrack`, `icmp`, `tcp`, `udp`, `ipvs`, `icmpv6` (if IPv6 is enabled in your kernel), `mark`, `multiport`, `rpfilter`, `sctp`, `ipvs` (if using `kube-proxy` in IPVS mode).
+  - At least the following actions: `REJECT`, `ACCEPT`, `DROP`, `LOG`.
+ 
+- IP sets support.
+- Netfilter Conntrack support compiled in (with SCTP support if using SCTP).
+- IPVS support if using `kube-proxy` in IPVS mode.
```

Comment:
This feature is not in CE yet (i.e. not tested or doc'd). Only in OS. Should we make it OS only?

Reply:
These docs only apply to Calico OSS, so I don't think we need to do anything here.